openlcs/libs/celery_helper.py

```diff
@@ -0,0 +1,23 @@
+
+priority_str_kwargs_map = {
+    "high": {},
```

Comment:
value is empty for high, it will use some default setting?

Reply:
empty is equivalent to priority=0, queue='celery'.

Reply:
ok, got it